packages/react/src/components/Tile/Tile-test.js

```diff
@@ -236,6 +237,19 @@ describe('Tile', () => {
   });
 
   describe('Renders expandable tile as expected', () => {
+    window.ResizeObserver = jest.fn(() => {
+      return {
+        observe: jest.fn(),
+        unobserve: jest.fn(),
+      };
+    });
+
```

Comment:
@joshblack for some reason (something about the async nature of window.ResizeObserver?) wrapping this assignment in a `beforeEach` block causes ResizeObserver to be undefined and error. If I'm not mistaken though the `afterAll` reset below should be correctly resetting the observer back to window default, right?